webapp/src/utils/index.tsx

```diff
@@ -175,6 +175,19 @@ const getFormattedFilters = (filters: FiltersData[]) => {
     return formattedFilters;
 };
 
+const getFiltersList = (formattedFilters: string): FiltersData[] => {
+    if (!formattedFilters) {
+        return [];
+    }
+
+    const filters: FiltersData[] = [];
+    JSON.parse(formattedFilters, (filterType, filterValue) => {
+        filters.push({filterType, filterValue});
+    });
+
+    return filters.slice(0, -1);
```

Comment:
Why are you removing the last element?

Reply:
to remove an extra element that is added to the list

Reply:
What is the extra element that is added to the list?

Reply:
it is removed in the following PR and the element is key-value pair of `""`, `{}`